src/components/ui/DialogUsageExample.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * This file is used as an example for the Dialog component.
+ * Once you're done with the example, you can delete this file.
+ */
+'use client';
+
+import { Button, Dialog } from '@/components/ui';
```

Comment:
Wildcard import should not be used if possible.
[Reference](https://stackoverflow.com/a/43569378)

I propose something like this.

```
import { Button } from '@/components/ui';
import { Body, Content, Footer, Header, Title } from '@/components/ui/Dialog';

import { Close, Root, Trigger } from '@radix-ui/react-dialog';
```

then you can write each node like `<Body>` instead of `<Dialog.Body>`

Reply:
As a result of the discussion, we concluded to avoid using wildcard imports as much as possible